processor/memorylimiterprocessor/factory_test.go

```diff
@@ -41,7 +41,7 @@ func TestCreateProcessor(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, tp)
 	// test if we can shutdown a monitoring routine that has not started
-	assert.ErrorIs(t, tp.Shutdown(context.Background()), errShutdownNotStarted)
+	assert.Error(t, tp.Shutdown(context.Background()))
```

Comment:
Please keep the error type check 
```suggestion
	assert. ErrorIs(t, tp.Shutdown(context.Background()), memorylimiter.ErrShutdownNotStarted)
```

Reply:
returned

Reply:
This one is still not updated

Reply:
sorry, missed that